language_specification/execution.md

```diff
@@ -0,0 +1,15 @@
+# Execution
+Execution options are open in each request.  This section describes the supported operations for core and each individual controller implementation.
```

Comment:
The meaning of the first sentence is confusing. Maybe "...open-ended for each request"